src/Security/Security.php

```diff
@@ -399,16 +454,14 @@ public function LoginForm()
      *
      * @todo Check how to activate/deactivate authentication methods
      */
-    public function GetLoginForms()
+    public function getLoginForms()
     {
-        $forms = array();
-
-        $authenticators = Authenticator::get_authenticators();
-        foreach ($authenticators as $authenticator) {
-            $forms[] = $authenticator::get_login_form($this);
-        }
-
-        return $forms;
+        return array_map(
+            function ($authenticator) {
+                return $authenticator->getLoginHandler($this->Link())->handleRequest($this->getRequest(), DataModel::inst());
```

Comment:
I think delegateToHandlers handles that now, I can't find anywhere where it's still needed, but we may have to look at re-enabling it.